policyengine_us/variables/gov/states/co/tax/income/credits/income_qualified_senior_housing/co_income_qualified_senior_housing_eligible.py

```diff
@@ -0,0 +1,64 @@
+from policyengine_us.model_api import *
+
+
+class co_income_qualified_senior_housing_eligible(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Eligible for Colorado Income Qualified Senior Housing Income Tax Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://tax.colorado.gov/income-qualified-senior-housing-income-tax-credit",
+        "https://tax.colorado.gov/sites/tax/files/documents/DR_104_Book_2022.pdf#page=17",
+    )
+    defined_for = StateCode.CO
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.co.tax.income.credits.income_qualified_senior_housing
+
+        filing_status = tax_unit("filing_status", period)
+        filing_statuses = filing_status.possible_values
+        single = filing_status == filing_statuses.SINGLE
+        joint = filing_status == filing_statuses.JOINT
+        hoh = filing_status == filing_statuses.HEAD_OF_HOUSEHOLD
+        widow = filing_status == filing_statuses.WIDOW
+        separate = filing_status == filing_statuses.SEPARATE
+
+        age_head = tax_unit("age_head", period)
+        age_spouse = select(
```

Comment:
we can just use tax_unit("age_spouse", period) - as in the single, hoh, and widow case there is no spouse present anyways